src/WebSdk/Publish/Targets/TransformTargets/Microsoft.NET.Sdk.Publish.TransformFiles.targets

```diff
@@ -54,6 +54,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
         And '$(PublishIISAssets)' != 'false'"
         TargetPath="$(TargetPath)"
         PublishDir="$(PublishIntermediateOutputPath)"
+        TargetFrameworkIdentifier="$(TargetFrameworkIdentifier)"
```

Comment:
This whole task/target is only applicable for ASP.NET Core projects. If the intent here is to not run the target for .NET Framework converted SDK projects, then I would add the logic directly in the target here and not change anything in the task.